README.md

```diff
@@ -20,9 +20,12 @@ In Maven, make your project section look like this:
 
   <repositories>
     <repository>
-      <id>liveramp-repositories</id>
-      <name>Liveramp Repositories</name>
-      <url>http://repository.liveramp.com/artifactory/liveramp-repositories</url>
+      <id>maven-snapshots</id>
+      <url>http://oss.sonatype.org/content/repositories/snapshots</url>
```

Comment:
Once we do a proper release to central, we should update this section to reference it instead.

Reply:
yep, will do -- think there's a separate and slightly longer discussion about versioning to figure out here, so prefer to leave it as-is for now.